classes/Utils.java

```diff
@@ -0,0 +1,18 @@
+package classes;
+
+public class Utils {
+
+    protected static Shirt[] addToArray(Shirt[] array, Shirt newItem) {
+        Shirt[] newArray = new Shirt[array.length + 1];
+        for (int i = 0; i < array.length; i++) {
+            newArray[i] = array[i];
+        }
+        newArray[array.length] = newItem;
+        return newArray;
+    }
+
+    public static boolean isClose(double value, double target, double tolerance) {
```

Comment:
```suggestion
    public static boolean isNear(double value, double target, double tolerance) {
```

this is how this func called in our code, is close if fine to but its better to now our conventions so i give this comment